src/checkout/CheckoutPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {FC} from 'react'
+
+// Components
+import CheckoutForm from 'src/checkout/CheckoutForm'
+import SuccessOverlay from 'src/checkout/SuccessOverlay'
+import CheckoutProvider from 'src/checkout/context/checkout'
+
+const CheckoutV2: FC = () => {
+  return (
+    <CheckoutProvider>
+      <React.Fragment>
```

Comment:
why use the fragment at all? get it out of here. also... stop typing fragment you weirdos. `<></>` is way better